atomic_reactor/plugins/pre_check_and_set_platforms.py

```diff
@@ -68,19 +68,29 @@ def run(self):
                     return set(override_platforms)
         else:
             platforms = get_orchestrator_platforms(self.workflow)
-            self.log.info("No koji platforms. User specified platforms are %s", sorted(platforms))
+            user_platforms = sorted(platforms) if platforms else None
+            self.log.info("No koji platforms. User specified platforms are %s", user_platforms)
 
         if not platforms:
             raise RuntimeError("Cannot determine platforms; no koji target or platform list")
 
         # Filter platforms based on clusters
         enabled_platforms = []
+        defined_but_disabled = []
         for p in platforms:
             if self.reactor_config.get_enabled_clusters_for_platform(p):
                 enabled_platforms.append(p)
             else:
-                self.log.warning(
-                    "No cluster found for platform '%s' in reactor config map, skipping", p)
+                if self.reactor_config.cluster_defined_for_platform(p):
```

Comment:
this could be changed to elif statement, and therefore the above else could be removed.

Reply:
ok

Reply:
fixed